packages/toolkit/src/view/knowledge/components/notifications/InsufficientStorageNotification.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, Icons } from "@instill-ai/design-system";
```

Comment:
@thewbuk "use client"

Please scan through all your basic level component, and add "use client" on them